install/upgrade_to_2.13.php

```diff
@@ -138,6 +133,103 @@ protected function addDashboardVisibility() {
       $table = 'glpi_plugin_formcreator_entityconfigs';
       $this->migration->addField($table, 'is_dashboard_visible', 'integer', ['after' => 'is_search_visible', 'value' => '-2']);
 
-      $this->migration->addPostQuery("UPDATE glpi_plugin_formcreator_entityconfigs SET `is_dashboard_visible`=1 WHERE `id`=0");
+      $this->migration->addPostQuery("UPDATE `glpi_plugin_formcreator_entityconfigs` SET `is_dashboard_visible`=1 WHERE `id`=0");
+   }
+
+   protected function migrateEntityConfig() {
+      global $DB;
+
+      $table = 'glpi_plugin_formcreator_entityconfigs';
+
+      $this->migration->addField($table, 'entities_id', 'int unsigned not null default 0', ['after' => 'id']);
+      $this->migration->migrationOneTable($table);
+      $this->migration->addKey($table, 'entities_id', 'unicity', 'UNIQUE');
+      $DB->queryOrDie("UPDATE `$table` SET `entities_id`=`id`");
+   }
+
+   protected function migrateFkToUnsignedInt() {
```

Comment:
this method updates many columns which are not handled by the GLPI FK migration tool. Most of these columns don't maatch the FK naming convention because they contein either an ID either a number to designate a business logic process.